common-lib/src/main/scala/com/gu/mediaservice/lib/metadata/ImageMetadataConverter.scala

```diff
@@ -7,6 +7,23 @@ import scala.util.Try
 
 import com.gu.mediaservice.model.{FileMetadata, ImageMetadata}
 
+object SupplierCategory extends Enumeration {
+  type SupplierCategory = Value
+  val Sport = Value("Sport")
+  val Entertainment = Value("Entertainment")
+  val Unknown = Value("Unknown")
+
+  // These category codes are now deprecated but still populated
+  // See: https://www.iptc.org/std/photometadata/documentation/GenericGuidelines/index.htm#!Documents/guidelineformappingcategorycodestosubjectnewscodes.htm
+  def create(category: String) = category match {
```

Comment:
Feels like an easy win to go beyond what they are just used to and provide more power, e.g. not just "sport" but "football", as a useful filter if you're covering a football match?

IIRC people also mentioned filtering out things like fashion, but I don't know if this was a genuine use case or a "maybe I'd do it". The point being that if they can't do it today with presents, they may not know the value of it.